datastore-translator/translator.go

```diff
@@ -18,21 +19,48 @@ import (
 )
 
 // ProtoMessageToDatastoreEntity will generate an Entity Protobuf that datastore understands
-func ProtoMessageToDatastoreEntity(src proto.Message, snakeCase bool, excludeFromIndex []string) (entity datastore.Entity, err error) {
+func ProtoMessageToDatastoreEntity(src proto.Message, snakeCase bool, excludeFromIndexName ...string) (entity datastore.Entity, err error) {
```

Comment:
I kinda missed that earlier, for explicitness sake, it wouldn't hurt if this argument is called ``excludeFromIndexOptionName`` or similar to make it clear that this expects extension option name.